Optimize (CQA) - Mng content - Managing CVs (#4897)
theforeman/foreman-documentation: guides/common/modules/proc_creating-a-rolling-content-view-by-using-web-ui.adoc

= Creating a rolling content view by using {ProjectWebUI}

[role="_abstract"]

> Create a rolling content view to give hosts continuous access to the latest synchronized content without publishing new versions.

Reviewer: ```suggestion
Create a rolling content view to give hosts continuous access to the latest synchronized content without publishing new content view versions.
```

.Prerequisites

> * At least one repository is synchronized to your organization.

Reviewer 2: The repositories don't have to by synced, they could have content imported or uploaded, but I think this may be pedantic. I wouldn't expect a user to assume these rolling content views are only for repositories with an upstream URL.

Author: Would this be clear enough? @ianballou @maximiliankolb 
```suggestion
* At least one populated repository exists in your organization.
```

Reviewer 2: That works for me!

Reviewer: Yes, alternative: "At least one repository exists in your organization." Your call; either way works for me.
